packages/action-group/package.json

```diff
@@ -58,7 +58,7 @@
         "lit-html"
     ],
     "dependencies": {
-        "@lit-labs/observers": "^2.0.0",
+        "@lit-labs/observers": "^2.0.2",
```

Comment:
I only made this change to ensure that all packages within SWC were pulling in the same version of `observers`; the others were on `2.0.2` already.

The reason why I went down this path at all in this otherwise-unrelated PR is that the discrepancy was causing `yarn link` to fail when linking UEC and SWC for testing purposes; making this change allowed `yarn link` to work! 🎉

We could consider bumping everything to the latest version, but that seemed like a truly unrelated change best done in a separate PR.

Reply:
Thanks for the context. Let us keep this as it is - I was mostly wondering if you had faced other upgrade-related conflicts or difficulties.